Marcio/warp content strategy (#2834)
cloudflare/cloudflare-docs: products/warp-client/src/content/warp-modes.md

---
order: 1
pcx-content-type: reference
---

# WARP modes

> The WARP client has several modes, to better suit different connection needs:

Reviewer: ```suggestion
The WARP client has several modes to better suit different connection needs:
```

## 1.1.1.1

1.1.1.1 is Cloudflare’s public DNS resolver. It offers a fast and private way to browse the Internet. It also offers an encrypted service through DNS over HTTPS (DoH) or DNS over TLS (DoT) for increased security and privacy.

Read more about 1.1.1.1 on our blog post [Announcing 1.1.1.1: the fastest, privacy-first consumer DNS service](https://blog.cloudflare.com/announcing-1111/) and [Introducing 1.1.1.1 for Families](https://blog.cloudflare.com/introducing-1-1-1-1-for-families/)

## 1.1.1.1 with WARP

> The WARP application uses [BoringTun](https://blog.cloudflare.com/boringtun-userspace-wireguard-rust/) to encrypt the traffic from your device and send it directly to Cloudflare’s edge. This ensures your connection is secure and private, and prevents third-parties from accessing your traffic. If the site you are visiting is already a Cloudflare customer, the content is immediately sent to your device. If not, Cloudflare uses our global network of data centers to devise the shortest path to the site.

Reviewer: ```suggestion
The WARP application uses [BoringTun](https://blog.cloudflare.com/boringtun-userspace-wireguard-rust/) to encrypt the traffic from your device and send it directly to Cloudflare’s edge network. This ensures your connection is secure and private and prevents third parties from accessing your traffic. If the site you are visiting is already a Cloudflare customer, the content is immediately sent to your device. If not, Cloudflare uses our global network of data centers to devise the shortest path to the site.
```

Read more about WARP in our blog post [Introducing WARP: Fixing Mobile Internet Performance and Security](https://blog.cloudflare.com/1111-warp-better-vpn/).

<Aside type='warning' header="Warning">
 

>  WARP does not provide anonymity and is not designed to prevent servers you communicate with from identifying you. WARP also does not allow you to pretend to be accessing the Internet from a different country than the one you are currently in.

Reviewer: ```suggestion
WARP does not provide anonymity and is not designed to prevent servers you communicate with from identifying you. WARP also does not allow you to access the Internet from a different country than the one you are currently in.
```

Author: Your suggestion doesn't really reflect what we should say here. It's not that WARP doesn't allow a customer to access the Internet from a different country, but rather it doesn't allow someone to pretend to be in a different country that they are.

</Aside>

## WARP via Local Proxy

> Currently, this mode is available on desktop clients only. When WARP is configured as a local proxy, only the applications that you configure to use the proxy (HTTPS or SOCKS5) will have their traffic sent through WARP. This allows you to pick and choose which traffic is encrypted — for instance, your web browser or a specific app. Everything else will not be encrypted and will be sent over a regular Internet connection.

Reviewer: ```suggestion
Currently, this mode is available on desktop clients only. When WARP is configured as a local proxy, only the applications that you configure to use the proxy (HTTPS or SOCKS5) will have their traffic sent through WARP. This allows you to pick and choose which traffic is encrypted — for example, your web browser or a specific application. Everything else will not be encrypted and will be sent over a regular Internet connection.
```

Because this feature restricts WARP to just applications configured to use the local proxy, leaving all other traffic unencrypted over the Internet by default, we have hidden it in the **Advanced** menu. To turn it on:

> 1. Navigate to **Preferences** > **Advanced** and click **Configure Proxy**.

Reviewer: ```suggestion
1. Navigate to **Preferences** > **Advanced** and select **Configure Proxy**.
```

1. On the window that opens, check the box and configure the port you want to listen on.

This will enable the **WARP via Local Proxy** option in the **WARP Settings** menu.

## WARP+

While WARP is able to take advantage of the many Cloudflare data centers around the world to give you a more private and robust connection, WARP+ improves on that with intelligent routing. Leveraging the same technology that powers [Argo Smart Routing](https://www.cloudflare.com/products/argo-smart-routing/), WARP+ will route your traffic around congested Internet routes and improve overall end-to-end performance.
 
## WARP Unlimited

> WARP Unlimited is our monthly subscription offering for WARP+. For the time being, WARP Unlimited can only be purchased via iOS and Android devices.

Reviewer: ```suggestion
WARP Unlimited is our monthly subscription offering for WARP+. Currently, WARP Unlimited can only be purchased via iOS and Android devices.
```
